docs/userGuide.md

```diff
@@ -8,20 +8,20 @@ OMA Lightweight M2M IoT Agent: User and Development Guide
 * [Development](#development)
 
 
-# <a name="overview"> Overview </a>
+#  Overview
 The Lightweight M2M IoT Agent is a standard Fiware IoT Agent that implements the bridge of the OMA Lightweight M2M protocol
-with the internal protocol for the FIWARE components (OMA NGSI). This IoT Agent is based in the public 
-Node.js IoT Agent Library, where more information can be found about what the IoT Agents are and their different APIs. 
+with the internal protocol for the FIWARE components (OMA NGSI). This IoT Agent is based in the public
+Node.js IoT Agent Library, where more information can be found about what the IoT Agents are and their different APIs.
 
 This project has, then, two APIs:
-* The South Bound (LWM2M): information about it can be found in the [OMA Lightweight M2M](http://openmobilealliance.org/iot/lightweight-m2m-lwm2m) official page. Information about the subset of Lightweight M2M already supported can be found in the [LWM2M Library for Node.js](https://github.com/telefonicaid/lwm2m-node-lib) we are using.
-* The North Bound Administration API: all the IoT Agents share a single Administration API, and it can be found in the [Node.JS IoT Agent Library Documentation](https://github.com/telefonicaid/iotagent-node-lib).
-* The North Bound NGSI API: information about the northbound NGSI mapping can be obtained in the same Node.JS IOTA Library documentation.
- 
+* The API for traffic south of the IoT Agent (LWM2M): information about it can be found in the [OMA Lightweight M2M](http://openmobilealliance.org/iot/lightweight-m2m-lwm2m) official page. Information about the subset of Lightweight M2M already supported can be found in the [LWM2M Library for Node.js](https://github.com/telefonicaid/lwm2m-node-lib) we are using.
+* The North Port Administration API: all the IoT Agents share a single Administration API, and it can be found in the [Node.JS IoT Agent Library Documentation](https://github.com/telefonicaid/iotagent-node-lib).
+* The API for traffic north of the IoT Agent (NGSI): information about the North Port NGSI mapping can be obtained in the same Node.JS IOTA Library documentation.
+
 You will find examples and more detailed information in the Getting Started howtos below.
- 
-# <a name="gettingstarted"> Getting started </a>
-This document links a set of howtos oriented to give a quick step-by-step example on how to use the agent with 
+
+# Getting started
```

Comment:
Now index link to this subsection does not work

Reply:
Fixed - c2bb99a - Updated ToC